release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,265 @@ toc::[]
+** Retrieval by using the `props` fields will be removed in version a future release.
+** Until removal, retrieval by using the `props` field will work if the result is unambiguous (no more than one group is found with the `props` field).
+* `/v1/cves/suppress` and `/v1/cves/unsuppress` have been deprecated and will be removed in a future release. After these are removed:
+** Use `/v1/imagecves/suppress` and `/v1/imagecves/unsuppress` to snooze and unsnooze image vulnerabilities.
+** Use `/v1/nodecves/suppress` and `/v1/nodecves/unsuppress` to snooze and unsnooze node and host vulnerabilities.
+** Use `/v1/clustercves/suppress` and `/v1/clustercves/unsuppress` to snooze and unsnooze platform (Kubernetes, Istio, and {ocp}) vulnerabilities.
+* Permission `ClusterCVE` is deprecated and will be superseded by the existing permission `Cluster`. 
+* `Label` and `Annotation` search options are deprecated and will be removed in the {product-title-short} 3.73 release. They will be replaced by the search options listed in the following table.
+
+.Search options
+[cols="1,1,1",options="header"]
+|====
+|Resource |Deprecated search option | New search option 
+
+| Node
+| Label
+| Node Label
+
+| Node
+| Annotation
+| Node Annotation
+
+| Namespace
+| Label
+| Namespace Label
+
+| Deployment
+| Label
+| Deployment Label
+
+| ServiceAccount
+| Label
+| Service Account Label
+
+| ServiceAccount
+| Annotation
+| Service Account Annotation
+
+| K8sRole
+| Label
+| Role Binding Label
+
+| K8sRoleAnnotation
+| Annotation
+| Role Binding Annotation
+
+|====
+
+[id="psp-changes"]
+=== PodSecurityPolicy (PSP) objects and Kubernetes 1.25
+
+The following changes were made in {product-title-short} to support the removal of PSP objects in Kubernetes 1.25:
+
+* PSP objects are created by default in installations for backward compatibility with Kubernetes versions prior to 1.25. However, you can manually disable PSP creation by setting `system.enablePodSecurityPolicies = false` in the Helm chart. 
+* Beginning with {product-title-short} version 3.71, auto-sensing has been added to {product-title-short} Helm charts. If you install {product-title-short} using the Operator or Helm charts, and {product-title-short} detects a Kubernetes version of 1.25 or later, it will not install PSPs. If you are using the `roxctl` CLI to install {product-title-short}, you need to disable PSP usage by setting the `--enable-pod-security-policies` flag to `false` for the `roxctl central generate` and `roxctl sensor generate` commands.
+
+Kubernetes users must disable the admission controller plugin for PSPs before upgrading to Kubernetes version 1.25. 
+
+
+[id="upcoming-api-changes"]
+=== Notice of upcoming API changes
+
+* CSV export: Beginning in the {product-title-short} 3.73 release, the CSV export `API /api/vm/export/csv` will require the `CVE Type` filter as part of the input query parameter. Requests that do not have the filter will give an error. Examples:
+** `CVE Type:NODE_CVE`
```

Comment:
```suggestion
** `CVE_Type:NODE_CVE`
```

Reply:
I believe this was done intentionally to show that you can use either of these options: `CVE Type`, `CVE_Type`, `CVE_TYPE`. @md2119 @charmik-redhat is that correct?

Reply:
The correct option is `CVE Type`, ACS search label. It seems correctly captured above but has 3 variations in `Example` section. I recommend adding: 
```
Supported values for `CVE Type` are `IMAGE_CVE`, `K8S_CVE`, `ISTIO_CVE`, `NODE_CVE`, and `OPENSHIFT_CVE`.
```